tests/migrations/test_writer.py

```diff
@@ -492,6 +499,33 @@ def upload_to(self):
         with self.assertRaisesMessage(ValueError, 'Could not find function upload_to in migrations.test_writer'):
             self.serialize_round_trip(TestModel2.thing)
 
+    def test_serialize_choices_iterator_should_unroll(self):
+        """An iterable marked to unroll should be converted to a list by
+        default."""
+        choices = [('a', 'A'), ('b', 'B')]
+
+        iter_choices = ImportableIterable()
+        iter_choices.unroll_in_migrations = True
```

Comment:
we were not sure if it was covered, it was also as a way to see if the patch was still working, maybe i should remove?